src/core/settings.c

```diff
@@ -144,6 +144,12 @@ QuicSettingsSetDefault(
     if (!Settings->IsSet.HyStartEnabled) {
         Settings->HyStartEnabled = QUIC_DEFAULT_HYSTART_ENABLED;
     }
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
```

Comment:
You don't need all these checks in the core code, just msquic.h. This is **always** defined for core.